examples/gno.land/r/demo/boards2/render.gno

```diff
@@ -3,11 +3,18 @@ package boards2
 import (
 	"strconv"
 
+	"gno.land/p/demo/avl/pager"
 	"gno.land/p/demo/mux"
 )
 
+const (
+	defaultPageSize   = 50
+	threadRenderDepth = 5
+)
+
 func Render(path string) string {
-	router := mux.NewRouter()
+	// TODO: remove this when query strings PR will be merged: https://github.com/gnolang/gno/pull/2594
+	router := newWrappedRouter()
```

Comment:
Another approach could also be something like:

```go
router := wrapRouter(mux.NewRouter())
```

Or instead of `wrapRouter` something like `fixRouter` or `patchRouter`. So when `mux` package is fixed to support query arguments we just remove the `router.gno` file and the function call, leaving `mux.NewRouter()`.

Reply:
Update, we won't try to fix the router here, so no wrapping is needed and `mux.NewRouter()` can be used directly.